ivy/functional/backends/jax/general.py

```diff
@@ -129,7 +152,7 @@ def gather(
     batch_dims: int = 0,
     out: Optional[JaxArray] = None,
 ) -> JaxArray:
-    axis = axis % len(params.shape)
+    axis = get_positive_axis_for_gather(axis, params.ndim)
     batch_dims = batch_dims % len(params.shape)
     ivy.utils.assertions.check_gather_input_valid(params, indices, axis, batch_dims)
```

Comment:
Also, please make sure to search for all the places that `ivy.gather` is being used in our codespace and change the `axis` value as needed. 
What I mean is, if `ivy.gather` is being called using the current default `axis` value, you will need to set it to `axis=-1` to make up for changing the default. I hope this makes sense!

Reply:
ok will fix it give 2 days